### PR TITLE
EIP-1013: Finalize all Constantinople hard-fork EIPs

### PR DESCRIPTION
**Pull request by 5chdn**
- move status to final
- add block number for mainnet

I'm not sure if meta EIPs need a last call here? They are just informational, aren't they?

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 1013 state from Draft to Final
 - EIP 1013 requires approval from one of (@nicksavers)

File: EIPS/eip-1013.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Cons
 - Codename: Constantinople
 - Aliases: Metropolis/Constantinople, Metropolis part 2
 - Activation:
-  - Block >= TBD on the Ethereum mainnet
-  - Block >= 4,230,000 on the Ropsten testnet
+  - `Block >= 7_080_000` on the Ethereum mainnet
+  - `Block >= 4_230_000` on the Ropsten testnet
 - Included EIPs:
```
Comment (axic) [inline EIPS/eip-1013.md]: I think this PR should also mark all these included EIPs as `Final`.

**Comment by 5chdn** [comment]
Can we get this merged soon? ref #1716

**Comment by 5chdn** [comment]
> There is a bug in 1052 and it should not be included in Constantinople in its current form.

Whatever the outcome is, Constantinople is final as it is active on Ropsten, Kovan, Rinkeby, Sokol and xDAI already. If we need to disable it, we will need a new upgrade. This does not affect EIP-1052 or 1013.

**Comment by holiman** [comment]
@fulldecent we answered, that submission is not an issue, it's a false positive. Feel free to share.

**Comment by axic** [comment]
@5chdn why did you close this? I think this one should be merged.

**Pull request closed without merging**